### ISSUE TITLE
Unify legacy-code based constants

### ISSUE THREAD

**Issue by jkirk5** [OpenMDAO/Aviary#363]
### Desired capability or behavior.

Aviary should have a single set of constants to use. These constants should be set based on a logical (and documented) source, such as ANSI standard constants or limited to a specific number of significant figures.

**Comment by jkirk5** [comment]
The constants file currently has takeoff/landing coefficients of friction. This is the wrong place for those variables - they are not universal physical constants. A user should be able to change them, so they should be variables in the hierarchy with default values that match what is currently in constants.

**Comment by xjjiang** [comment]
Should we hard code `6.687 lbm/galUS` as aviation fuel density? This is GASP default. On FLOPS side, the default is `6.7`.

**Comment by xjjiang** [comment]
> The constants file currently has takeoff/landing coefficients of friction. This is the wrong place for those variables - they are not universal physical constants. A user should be able to change them, so they should be variables in the hierarchy with default values that match what is currently in constants.

This is done in PR #1170.

**Comment by jkirk5** [comment]
> Should we hard code `6.687 lbm/galUS` as aviation fuel density? This is GASP default. On FLOPS side, the default is `6.7`.

Good point! That should also be a constant. I am planning an update to the way Aviary handles fuel, and I can add fuel densities to our constants as part of that issue!
